index.php

```diff
@@ -99,24 +108,24 @@ function getConfig($strPath)
 }
 if (ini_get('register_argc_argv') == 0) {
     echo "register_argc_argv is not turned on, we can't process command line arguments.\n";
-    exit(1);
+    //exit(1);
```

Comment:
Is there a way to read command line arguments without <code>register_argc_argv</code> extension?

Reply:
I use for my CLI scripts
```
foreach($argc as $some)
```